CustomApps/lyrics-plus/ProviderNetease.js

```diff
@@ -17,8 +17,10 @@ const ProviderNetease = (function () {
 		}
 
 		const album = Utils.capitalize(info.album);
-		let itemId = items.findIndex(val => Utils.capitalize(val.album.name) === album);
-		if (itemId === -1) itemId = 0;
+		let itemId = items.findIndex(
+			val => Utils.capitalize(val.album.name) === album || (info.duration + 1000 > val.duration && info.duration - 1000 < val.duration)
```

Comment:
This should be valid.
```suggestion
			val => Utils.capitalize(val.album.name) === album || Math.abs(info.duration - val.duration) < 1000
```